app/services/issuing_ca_service.rb

```diff
@@ -73,4 +76,10 @@ def self.allowed_host?(host)
     Rails.logger.info("CA Issuer Host Not Allowed: #{host}")
     false
   end
+
+  def self.certificate_store_issuers
+    CertificateStore.instance.certificates.map do |certificate|
+      ca_issuers_for_cert(certificate)
+    end.flatten.compact.uniq
```

Comment:
do we need the `.compact` here if `ca_issuers_for_cert` always returns some sort of array?

Reply:
yeah, it'll always be an array, but the arrays could contain nil, e.g. `[[nil, "piv.host.gov"], [nil]]`. It'll get flattened to `[nil, "piv.host.gov", nil]`, and the `compact` removes those nils.

Reply:
But what about the `    end.compact` line? woudn't that get rid of those intermediate `nils`?

Reply:
ohh, yep, I understand now. Will remove it.